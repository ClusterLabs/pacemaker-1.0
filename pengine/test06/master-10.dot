 digraph "g" {
"child_rsc1:0_monitor_0 node1" -> "probe_complete node1" [ style = bold]
"child_rsc1:0_monitor_0 node1" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:0_monitor_0 node2" -> "probe_complete node2" [ style = bold]
"child_rsc1:0_monitor_0 node2" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:0_monitor_11000 node1" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:0_post_notify_promote_0 node1" -> "rsc1_confirmed-post_notify_promote_0" [ style = bold]
"child_rsc1:0_post_notify_promote_0 node1" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:0_post_notify_start_0 node1" -> "rsc1_confirmed-post_notify_start_0" [ style = bold]
"child_rsc1:0_post_notify_start_0 node1" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:0_pre_notify_promote_0 node1" -> "rsc1_confirmed-pre_notify_promote_0" [ style = bold]
"child_rsc1:0_pre_notify_promote_0 node1" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:0_promote_0 node1" -> "child_rsc1:0_monitor_11000 node1" [ style = bold]
"child_rsc1:0_promote_0 node1" -> "rsc1_promoted_0" [ style = bold]
"child_rsc1:0_promote_0 node1" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:0_start_0 node1" -> "child_rsc1:0_monitor_11000 node1" [ style = bold]
"child_rsc1:0_start_0 node1" -> "child_rsc1:0_promote_0 node1" [ style = bold]
"child_rsc1:0_start_0 node1" -> "rsc1_running_0" [ style = bold]
"child_rsc1:0_start_0 node1" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:1_monitor_0 node1" -> "probe_complete node1" [ style = bold]
"child_rsc1:1_monitor_0 node1" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:1_monitor_0 node2" -> "probe_complete node2" [ style = bold]
"child_rsc1:1_monitor_0 node2" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:1_monitor_1000 node2" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:1_post_notify_promote_0 node2" -> "rsc1_confirmed-post_notify_promote_0" [ style = bold]
"child_rsc1:1_post_notify_promote_0 node2" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:1_post_notify_start_0 node2" -> "rsc1_confirmed-post_notify_start_0" [ style = bold]
"child_rsc1:1_post_notify_start_0 node2" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:1_pre_notify_promote_0 node2" -> "rsc1_confirmed-pre_notify_promote_0" [ style = bold]
"child_rsc1:1_pre_notify_promote_0 node2" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:1_start_0 node2" -> "child_rsc1:1_monitor_1000 node2" [ style = bold]
"child_rsc1:1_start_0 node2" -> "rsc1_running_0" [ style = bold]
"child_rsc1:1_start_0 node2" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:2_monitor_0 node1" -> "probe_complete node1" [ style = bold]
"child_rsc1:2_monitor_0 node1" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:2_monitor_0 node2" -> "probe_complete node2" [ style = bold]
"child_rsc1:2_monitor_0 node2" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:2_monitor_1000 node1" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:2_post_notify_promote_0 node1" -> "rsc1_confirmed-post_notify_promote_0" [ style = bold]
"child_rsc1:2_post_notify_promote_0 node1" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:2_post_notify_start_0 node1" -> "rsc1_confirmed-post_notify_start_0" [ style = bold]
"child_rsc1:2_post_notify_start_0 node1" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:2_pre_notify_promote_0 node1" -> "rsc1_confirmed-pre_notify_promote_0" [ style = bold]
"child_rsc1:2_pre_notify_promote_0 node1" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:2_start_0 node1" -> "child_rsc1:2_monitor_1000 node1" [ style = bold]
"child_rsc1:2_start_0 node1" -> "rsc1_running_0" [ style = bold]
"child_rsc1:2_start_0 node1" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:3_monitor_0 node1" -> "probe_complete node1" [ style = bold]
"child_rsc1:3_monitor_0 node1" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:3_monitor_0 node2" -> "probe_complete node2" [ style = bold]
"child_rsc1:3_monitor_0 node2" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:3_monitor_1000 node2" [ style=bold color="green" fontcolor="black"  ]
"child_rsc1:3_post_notify_promote_0 node2" -> "rsc1_confirmed-post_notify_promote_0" [ style = bold]
"child_rsc1:3_post_notify_promote_0 node2" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:3_post_notify_start_0 node2" -> "rsc1_confirmed-post_notify_start_0" [ style = bold]
"child_rsc1:3_post_notify_start_0 node2" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:3_pre_notify_promote_0 node2" -> "rsc1_confirmed-pre_notify_promote_0" [ style = bold]
"child_rsc1:3_pre_notify_promote_0 node2" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:3_start_0 node2" -> "child_rsc1:3_monitor_1000 node2" [ style = bold]
"child_rsc1:3_start_0 node2" -> "rsc1_running_0" [ style = bold]
"child_rsc1:3_start_0 node2" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:4_monitor_0 node1" -> "probe_complete node1" [ style = bold]
"child_rsc1:4_monitor_0 node1" [ style=bold color="green" fontcolor="black" ]
"child_rsc1:4_monitor_0 node2" -> "probe_complete node2" [ style = bold]
"child_rsc1:4_monitor_0 node2" [ style=bold color="green" fontcolor="black" ]
"probe_complete node1" -> "probe_complete" [ style = bold]
"probe_complete node1" [ style=bold color="green" fontcolor="black" ]
"probe_complete node2" -> "probe_complete" [ style = bold]
"probe_complete node2" [ style=bold color="green" fontcolor="black" ]
"probe_complete" -> "child_rsc1:0_start_0 node1" [ style = bold]
"probe_complete" -> "child_rsc1:1_start_0 node2" [ style = bold]
"probe_complete" -> "child_rsc1:2_start_0 node1" [ style = bold]
"probe_complete" -> "child_rsc1:3_start_0 node2" [ style = bold]
"probe_complete" -> "rsc1_start_0" [ style = bold]
"probe_complete" -> "rsc1_stop_0" [ style = bold]
"probe_complete" [ style=bold color="green" fontcolor="orange" ]
"rsc1_confirmed-post_notify_promote_0" -> "child_rsc1:0_monitor_11000 node1" [ style = bold]
"rsc1_confirmed-post_notify_promote_0" -> "child_rsc1:1_monitor_1000 node2" [ style = bold]
"rsc1_confirmed-post_notify_promote_0" -> "child_rsc1:2_monitor_1000 node1" [ style = bold]
"rsc1_confirmed-post_notify_promote_0" -> "child_rsc1:3_monitor_1000 node2" [ style = bold]
"rsc1_confirmed-post_notify_promote_0" [ style=bold color="green" fontcolor="orange" ]
"rsc1_confirmed-post_notify_start_0" -> "child_rsc1:0_monitor_11000 node1" [ style = bold]
"rsc1_confirmed-post_notify_start_0" -> "child_rsc1:1_monitor_1000 node2" [ style = bold]
"rsc1_confirmed-post_notify_start_0" -> "child_rsc1:2_monitor_1000 node1" [ style = bold]
"rsc1_confirmed-post_notify_start_0" -> "child_rsc1:3_monitor_1000 node2" [ style = bold]
"rsc1_confirmed-post_notify_start_0" -> "rsc1_pre_notify_promote_0" [ style = bold]
"rsc1_confirmed-post_notify_start_0" -> "rsc1_promote_0" [ style = bold]
"rsc1_confirmed-post_notify_start_0" [ style=bold color="green" fontcolor="orange" ]
"rsc1_confirmed-pre_notify_promote_0" -> "rsc1_promote_0" [ style = bold]
"rsc1_confirmed-pre_notify_promote_0" [ style=bold color="green" fontcolor="orange" ]
"rsc1_confirmed-pre_notify_start_0" -> "rsc1_start_0" [ style = bold]
"rsc1_confirmed-pre_notify_start_0" [ style=bold color="green" fontcolor="orange" ]
"rsc1_post_notify_promote_0" -> "child_rsc1:0_post_notify_promote_0 node1" [ style = bold]
"rsc1_post_notify_promote_0" -> "child_rsc1:1_post_notify_promote_0 node2" [ style = bold]
"rsc1_post_notify_promote_0" -> "child_rsc1:2_post_notify_promote_0 node1" [ style = bold]
"rsc1_post_notify_promote_0" -> "child_rsc1:3_post_notify_promote_0 node2" [ style = bold]
"rsc1_post_notify_promote_0" -> "rsc1_confirmed-post_notify_promote_0" [ style = bold]
"rsc1_post_notify_promote_0" [ style=bold color="green" fontcolor="orange" ]
"rsc1_post_notify_start_0" -> "child_rsc1:0_post_notify_start_0 node1" [ style = bold]
"rsc1_post_notify_start_0" -> "child_rsc1:1_post_notify_start_0 node2" [ style = bold]
"rsc1_post_notify_start_0" -> "child_rsc1:2_post_notify_start_0 node1" [ style = bold]
"rsc1_post_notify_start_0" -> "child_rsc1:3_post_notify_start_0 node2" [ style = bold]
"rsc1_post_notify_start_0" -> "rsc1_confirmed-post_notify_start_0" [ style = bold]
"rsc1_post_notify_start_0" [ style=bold color="green" fontcolor="orange" ]
"rsc1_pre_notify_promote_0" -> "child_rsc1:0_pre_notify_promote_0 node1" [ style = bold]
"rsc1_pre_notify_promote_0" -> "child_rsc1:1_pre_notify_promote_0 node2" [ style = bold]
"rsc1_pre_notify_promote_0" -> "child_rsc1:2_pre_notify_promote_0 node1" [ style = bold]
"rsc1_pre_notify_promote_0" -> "child_rsc1:3_pre_notify_promote_0 node2" [ style = bold]
"rsc1_pre_notify_promote_0" -> "rsc1_confirmed-pre_notify_promote_0" [ style = bold]
"rsc1_pre_notify_promote_0" [ style=bold color="green" fontcolor="orange" ]
"rsc1_pre_notify_start_0" -> "rsc1_confirmed-pre_notify_start_0" [ style = bold]
"rsc1_pre_notify_start_0" [ style=bold color="green" fontcolor="orange" ]
"rsc1_promote_0" -> "child_rsc1:0_promote_0 node1" [ style = bold]
"rsc1_promote_0" [ style=bold color="green" fontcolor="orange" ]
"rsc1_promoted_0" -> "rsc1_post_notify_promote_0" [ style = bold]
"rsc1_promoted_0" [ style=bold color="green" fontcolor="orange" ]
"rsc1_running_0" -> "rsc1_post_notify_start_0" [ style = bold]
"rsc1_running_0" -> "rsc1_promote_0" [ style = bold]
"rsc1_running_0" [ style=bold color="green" fontcolor="orange" ]
"rsc1_start_0" -> "child_rsc1:0_start_0 node1" [ style = bold]
"rsc1_start_0" -> "child_rsc1:1_start_0 node2" [ style = bold]
"rsc1_start_0" -> "child_rsc1:2_start_0 node1" [ style = bold]
"rsc1_start_0" -> "child_rsc1:3_start_0 node2" [ style = bold]
"rsc1_start_0" -> "rsc1_promote_0" [ style = bold]
"rsc1_start_0" -> "rsc1_running_0" [ style = bold]
"rsc1_start_0" [ style=bold color="green" fontcolor="orange" ]
"rsc1_stop_0" -> "rsc1_start_0" [ style = bold]
"rsc1_stop_0" [ style=bold color="green" fontcolor="orange"  ]
}
