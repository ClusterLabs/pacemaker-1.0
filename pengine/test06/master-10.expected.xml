<?xml version="1.0" encoding="UTF-8"?>
<cib admin_epoch="0" epoch="1" num_updates="1" dc-uuid="0" have-quorum="false" remote-tls-port="0" validate-with="pacemaker-1.0">
  <configuration>
    <crm_config><cluster_property_set id="cib-bootstrap-options"><nvpair id="nvpair" name="no-quorum-policy" value="ignore"/></cluster_property_set></crm_config>
    <nodes>
      <node id="uuid1" uname="node1" type="member">
        <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="master-child_rsc1:0" value="20"/><nvpair id="nvpair" name="master-child_rsc1:2" value="10"/></instance_attributes>
      </node>
      <node id="uuid2" uname="node2" type="member">
        <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="master-child_rsc1:1" value="10"/><nvpair id="nvpair" name="master-child_rsc1:3" value="20"/></instance_attributes>
      </node>
    </nodes>
    <resources>
      <master id="rsc1"><meta_attributes id="master_slave-rsc1.meta"><nvpair id="notify.meta.auto-43" name="notify" value="true"/><nvpair id="nvpair" name="resource-stickiness" value="1"/><nvpair id="nvpair" name="clone-max" value="5"/><nvpair id="nvpair" name="clone-node-max" value="2"/><nvpair id="nvpair" name="master-max" value="2"/><nvpair id="nvpair" name="master-node-max" value="1"/></meta_attributes><primitive id="child_rsc1" class="heartbeat" type="apache"><meta_attributes id="primitive-child_rsc1.meta"/>
	  <operations>
	    <op name="monitor" interval="1" id="mon-1"/>
	    <op name="monitor" interval="11" id="mon-2" role="Master"/>
	  </operations>
	</primitive></master>
    </resources>
    <constraints>
      <rsc_location id="master-node2" rsc="rsc1">
	<rule id="master-rule-node2" score="100" role="Master">
	  <expression id="expression" attribute="#uname" operation="eq" value="node1"/>
	</rule>
      </rsc_location>
    </constraints>
  </configuration>
  <status>
    <node_state id="uuid1" ha="active" uname="node1" crmd="online" join="member" expected="member" in_ccm="true">
      <transient_attributes id="transient_attributes">
        <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="probe_complete" value="true"/></instance_attributes>
      </transient_attributes>
    </node_state>
    <node_state id="uuid2" ha="active" uname="node2" crmd="online" join="member" expected="member" in_ccm="true">
      <transient_attributes id="transient_attributes">
        <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="probe_complete" value="true"/></instance_attributes>
      </transient_attributes>
    </node_state>
  </status>
</cib>
